monai/losses/dice.py

```diff
@@ -202,3 +197,7 @@ def forward(self, pred, ground, smooth=1e-5):
 
         f = (2.0 * intersection * w + smooth) / (denominator * w + smooth)
         return 1.0 - f.mean()  # final reduce_mean across batches and channels
+
+
+dice = Dice = DiceLoss
+generalized_dice = generalised_dice = GeneralizedDiceLoss
```

Comment:
I think we don't need to consider "s" instead "z" in some spelling, otherwise, there are many code lines related to this...

Reply:
the original paper and niftynet uses s, i’d like to keep this specific alias